docs/DropdownToolbar.md

```diff
@@ -0,0 +1,69 @@
+# Dropdown Toolbar
+
+The `<pxb-dropdown-toolbar>` component is used to display a toolbar with a dropdown.
+
+<div style="text-align:center">
+<img width="100%" style="max-width:600px" alt="Dropdown Toolbar" src="./images/dropdownToolbar.png">
```

Comment:
This image looks like it's still not the recommended spacing on the action items (see comment above).